x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/form_validation/downsample_interval.test.ts

```diff
@@ -0,0 +1,130 @@
+    },
+    {
+      name: `doesn't allow -1 for interval`,
+      value: '-1',
+      error: [i18nTexts.editPolicy.errors.numberGreatThan0Required],
+    },
+    {
+      name: `doesn't allow decimals for timing (with dot)`,
+      value: '5.5',
+      error: [i18nTexts.editPolicy.errors.integerRequired],
+    },
+    {
+      name: `doesn't allow decimals for timing (with comma)`,
+      value: '5,5',
+      error: [i18nTexts.editPolicy.errors.integerRequired],
+    },
+  ].forEach((testConfig: { name: string; value: string; error: string[] }) => {
+    (['hot', 'warm', 'cold'] as PhaseWithDownsample[]).forEach((phase: PhaseWithDownsample) => {
+      const { name, value, error } = testConfig;
+      test(`${phase}: ${name}`, async () => {
+        if (phase !== 'hot') {
+          await actions.togglePhase(phase);
+        }
+
+        await actions[phase].downsample.toggle();
+
+        // 1. We first set as dummy value to have a starting min_age value
+        await actions[phase].downsample.setDownsampleInterval('111');
+        // 2. At this point we are sure there will be a change of value and that any validation
+        // will be displayed under the field.
+        await actions[phase].downsample.setDownsampleInterval(value);
+
+        actions.errors.waitForValidation();
+
+        actions.errors.expectMessages(error);
+      });
+    });
+  });
+
+  test('should validate an interval is greater or multiple than previous phase interval', async () => {
+    await actions.togglePhase('warm');
+    await actions.togglePhase('cold');
+
+    await actions.hot.downsample.toggle();
+    await actions.hot.downsample.setDownsampleInterval('60', 'm');
+
+    await actions.warm.downsample.toggle();
+    await actions.warm.downsample.setDownsampleInterval('1', 'h');
+
+    actions.errors.waitForValidation();
+    actions.errors.expectMessages(
+      ['Must be greater than and a multiple of the hot phase value (60m)'],
+      'warm'
+    );
+
+    await actions.cold.downsample.toggle();
+    await actions.cold.downsample.setDownsampleInterval('90', 'm');
+    actions.errors.waitForValidation();
+    actions.errors.expectMessages(
+      ['Must be greater than and a multiple of the warm phase value (1h)'],
+      'cold'
+    );
+
+    // disable warm phase;
+    await actions.togglePhase('warm');
+    // TODO: there is a bug that disabling a phase doesn't trigger downsample validation in other phases,
+    // users can workaround it by changing the value
+    await actions.cold.downsample.setDownsampleInterval('120', 'm');
+    actions.errors.waitForValidation();
+    actions.errors.expectMessages([], 'cold');
+
+    await actions.cold.downsample.setDownsampleInterval('90', 'm');
+    actions.errors.waitForValidation();
+    actions.errors.expectMessages(
+      ['Must be greater than and a multiple of the hot phase value (60m)'],
+      'cold'
+    );
+  }, /* increase a timeout of this test as it could fail on ci*/ 12000);
```

Comment:
Out of curiosity - is there any more context you can provide here? Was the test failing without the increased timeout?

Reply:
It was failing with a 5-second timeout because it was a longer test. initially, I also implemented a frozen phase and tested it here. Since we removed the frozen phase, I also removed a timeout, as I expect it should fit in 5 seconds now.